packages/synapse-react-client/test/components/SynapseTable/SynapseTableUtils.test.ts

```diff
@@ -293,4 +294,26 @@ describe('Synapse Table Utilities tests', () => {
       expect(getFileColumnModelId(columnModels)).toEqual(undefined)
     })
   })
+
+  describe('isSortableColumn', () => {
+    it('throws a warning and returns false for an unknown type', () => {
+      const consoleWarnSpy = jest
+        .spyOn(console, 'warn')
+        .mockImplementation(() => {})
+
+      // Call under test
+      isSortableColumn('FAKE_COLUMN_TYPE' as ColumnTypeEnum)
+
+      expect(consoleWarnSpy).toHaveBeenCalled()
+      consoleWarnSpy.mockRestore()
+    })
+
+    it('explicitly handles all known types', () => {
+      const consoleWarnSpy = jest.spyOn(console, 'warn')
+      Object.keys(ColumnTypeEnum).forEach(key => {
+        isSortableColumn(key)
+      })
+      expect(consoleWarnSpy).not.toHaveBeenCalled()
+    })
```

Comment:
Also ensure that if we have a new type in the enum, we must explicitly declare it to be sortable or not.